Source/RapyutaSimulationPlugins/Private/Tools/RRROS2OdomPublisher.cpp

```diff
@@ -52,21 +52,21 @@ bool URRROS2OdomPublisher::GetOdomData(FROSOdometry& OutOdomData) const
 {
     const URobotVehicleMovementComponent* moveComponent =
         RobotVehicle.IsValid() ? RobotVehicle.Get()->RobotVehicleMoveComponent : nullptr;
-    if (moveComponent && moveComponent->OdomData.IsValid())
+    if (moveComponent)

```

Comment:
Just a note about `OdomData.IsValid()`, which [checks](https://github.com/rapyuta-robotics/rclUE/blob/devel/Source/rclUE/Public/Msgs/ROS2OdometryMsg.h#L57) `PoseCovariance, TwistCovariance` arrays for having been initialized with `36` elements.
So here removing this check may require adding the initialization in [FROSOdometry](https://github.com/rapyuta-robotics/rclUE/blob/update_autogen/Source/rclUE/Public/Msgs/ROS2OdometryMsg.h)?

Reply:
Got it. that should be handled by UE_tools autogen.
I added limitation/todo of UE_tools readme